lib/components/navigation/SideBar.tsx

```diff
@@ -30,19 +32,43 @@ const SideBar: FC<SideBarProps> = ({ open, onClose }) => {
     }
   });
 
+  const handleEditClick = () => {
+    setEditMode(!isEditMode);
+    setSelected('');
+  };
+
   const generateSiteLinks = () => {
     if (sites) {
       // TODO: remove hard-coded limit of 5 solar panel sites
-      return sites
-        .slice(0, 5)
-        .map((site, idx) => (
+      return sites.slice(0, 5).map((site, idx) => (
+        <div
```

Comment:
try giving this div w-full, and give the radio button div flex-1 and the dashboard link div flex-shrink-1

that might fix the thing where the cards are expanding out, making the sidebar bigger

Reply:
doesn't work :(